test/flint/fq_default_extended-test.jl

```diff
@@ -287,4 +287,12 @@ end
     end
     @test divexact(a, one(FF)) == divexact(FF(a), one(FF))
   end
+
+  F = Nemo._GF(2)
+  Fx, x = F["x"]
+  FF, = Nemo._residue_field(x - 2)
```

Comment:
```suggestion
  F = Nemo._GF(3)
  Fx, x = F["x"]
  FF, = Nemo._residue_field(x - 2)
```
The polynomial x-2 over char 2 (and thus equal to x) was probably not what you wanted to test here